org-roam.el

```diff
@@ -1113,9 +1113,10 @@ into a digraph."
 	  (insert "}")
 	  (buffer-string)))
 
-(defun org-roam-show-graph ()
-  "Displays the generated Org-roam graph using `org-roam-graph-viewer'."
-  (interactive)
+(defun org-roam-show-graph (prefix)
```

Comment:
&optional prefix

Reply:
I appreciate the improvements! FWIW, I used `org-roam-insert` as a style reference for the prefix argument, so perhaps we should change that, too.